index.html

```diff
@@ -42,16 +42,15 @@
       Do you see any Teletubbies in here? Do you see a slender plastic tag clipped to my shirt with my name printed on it? Do you see a little Asian child with a blank expression on his face sitting outside on a mechanical helicopter that shakes when you put quarters in it? No? Well, that's what you see at a toy store. And you must think you're in a toy store, because you're here shopping for an infant named Jeb.
 
       <!-- end slipsum code --></p>
-
+<h1>Employment History</h1>
       <section id="jobHistories">
         <article class="listJobs" data-category="cat">
-          <h1>Employment History:</h1>
           <div class="resume">
-            <h4 class=".jobTitle"></h4>
-            <h4 class=".empName"></h4>
-            <h4 class=".empLoc"></h4>
-            <h4 class=".dateEmp"></h4>
-            <p class=".respon"></p>
+            <h4 class="jobTitle"></h4>
```

Comment:
We looked at Davids code in atom . and fixed the issue of having a reversal of class in html on this end with a . and then having to put that into the .js file.